server/routes/what-species-expert.route.js

```diff
@@ -62,51 +65,58 @@ const handlers = {
         RedisKeys.WHAT_SPECIES,
         payload.whatSpecies
       )
-      return h.redirect(Paths.WHAT_TYPE_OF_ITEM_IS_IT)
 
-    // If not sure display a screen giving the user the option to proceed or end their journey
-    } else if (payload.whatSpecies === twoOrMoreOption) {
-      await RedisService.set(
-        request,
-        RedisKeys.WHAT_SPECIES,
-        payload.whatSpecies
-      )
-      return h.redirect(Paths.WHAT_TYPE_OF_ITEM_IS_IT)
+      const useChecker = await getUseChecker(request)
+
+      if (useChecker) {
+        return h.redirect(Paths.SELLING_TO_MUSEUM)
+      } else {
+        return h.redirect(Paths.HOW_CERTAIN)
+      }
 
     // If not sure display a screen giving the user the option to proceed or end their journey
-    } else if (payload.whatSpecies === notSureOption) {
+    } else {
       await RedisService.set(
         request,
         RedisKeys.WHAT_SPECIES,
         payload.whatSpecies
       )
       return h.redirect(Paths.OPTION_TO_PROCEED)
-
-    // If none of these then end the journey
-    } else {
-      await RedisService.delete(request, RedisKeys.WHAT_SPECIES)
-
-      return h.redirect(Paths.DO_NOT_NEED_SERVICE)
     }
   }
 }
 
 const _getContext = async request => {
   return {
-    pageTitle: 'What species of ivory does your item contain?',
+    pageTitle: 'Does your item contain banned ivory?',
     speciesItems: await _getSpeciesItems(request),
     items: await _getOptions(request),
     eligibilityCheckerUrl: Paths.CONTAIN_ELEPHANT_IVORY,
     guidanceUrl: Urls.GOV_UK_TOP_OF_MAIN
   }
 }
 
+const getUseChecker = async request => {
+  const useChecker = await RedisService.get(request, RedisKeys.USE_CHECKER)
+
+  if (useChecker) {
+    await RedisService.set(request, RedisKeys.USED_CHECKER, true)
+  }
+
+  RedisService.delete(request, RedisKeys.USE_CHECKER)
```

Comment:
It concerns me that the underlying asynchronous operation isn't being awaited here (looks like elsewhere in the code base as well)

Reply:
Its actually not an async function and sonarcloud picked up on it
<img width="855" alt="image" src="https://github.com/DEFRA/ivory/assets/141018006/314600b1-5418-4472-8998-4638830e4a67">

Reply:
That underlying `client.del()` call returns a Promise - the Redis call is asynchronous but for some reason that `delete` method isn't returning the Promise.

It's highly unlikely to be causing any issues. But with random problems happening on the service, having operations that are not being properly handled makes me nervous.

As I say, it looks like this issue exists elsewhere in the code base, although thankfully there aren't many deletes happening during the journey.

I'd just rather not add to the pile if we can help it.

Reply:
Have applied above